js/modules/Forms/ServiceCatalogController.js

```diff
@@ -209,4 +233,9 @@ export class GlpiFormServiceCatalogController
     {
         return document.querySelector("[data-glpi-service-catalog-items]");
     }
+
+    getTemplateForSortSelect(data) {
+        const icon = this.sort_icons[data.id];
+        return $(`<span class="d-block w-full" title="${data.text}" aria-label="${data.text}"><i class="${icon}"></i></span>`);
```

Comment:
Could you check if it can cause issues when `data.text` contains double quotes ? If it cause issues, you should use `_.escape()` to prevent XSS issues.